doctrine/doctrine-bundle/2.8/manifest.json

```
@@ -11,7 +11,8 @@
         "#2": "IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml",
         "#3": "",
         "#4": "DATABASE_URL=\"sqlite:///%kernel.project_dir%/var/data.db\"",
-        "#5": "DATABASE_URL=\"mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=8&charset=utf8mb4\"",
+        "#5": "DATABASE_URL=\"mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=mariadb-10.11&charset=utf8mb4\"",
+        "#6": "DATABASE_URL=\"mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=8&charset=utf8mb4\"",
```

Oracle is rolling out new features in MySQL 8.0.X releases nowadays, so it's a good practice to specify the full version.

```suggestion
        "#6": "DATABASE_URL=\"mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=8.0.32&charset=utf8mb4\"",
```

---

Always fan of full version 👍 . Should we specifiy it also for postgresql?

---

> Should we specifiy it also for postgresql?

We can do this for the sake of consistency, but since Postgres implements a much saner versioning policy, it's not really necessary.